pytest/common_lib/shared/mpc_cluster.py

```diff
@@ -68,16 +68,16 @@ def reset_mpc_data(self, node_idxs: List[int]):
         for node_idx in node_idxs:
             self.mpc_nodes[node_idx].reset_mpc_data()
 
-    def __init__(self, main: NearAccount, secondary: NearAccount):
+    def __init__(self, main: NearAccount, secondary: Optional[NearAccount] = None):
         self.mpc_nodes: List[MpcNode] = []
         # Note: Refer to signing schemas and key resharing
         self.next_participant_id: int = 0
         # Main account where Chain Signatures contract is deployed
         self.contract_node = main
         # In some tests we may need another CS contract.
-        self.secondary_contract_node = secondary
+        self.secondary_contract_node = secondary or main
```

Comment:
can't we set this to `None` if `secondary` is `None`? Otherwise, we risk accidentally overriding the contract.